test/BlockbookEthereum.test.ts

```diff
@@ -31,11 +32,24 @@ describe('BlockbookEthereum', () => {
       expect(await bb.getTxSpecific(TXID)).toBeDefined()
     })
   })
+
   describe('getAddressDetails', () => {
     it('succeeds', async () => {
       expect(await bb.getAddressDetails(ADDRESS)).toBeDefined()
     })
   })
+
+  describe('getAddressDetails2', () => {
+    it('succeeds', async () => {
+      expect(await bb.getAddressDetails('0x176366cFD97885245fAEA72f8cB6951e52655Adf')).toBeDefined()
+    })
+  })
+
+  describe('getAddressDetails - faa.st wallet', () => {
+    it('succeeds', async () => {
+      expect(await bb.getAddressDetails('0x94fe3ad91dacba8ec4b82f56ff7c122181f1535d')).toBeDefined()
+    })
+  })
```

Comment:
@dylanseago I was able to simulate the other breaking test.

```ts

  ● BlockbookEthereum › getAddressDetails - faa.st wallet › succeeds

    TypeError: Invalid type - Expected type "ERC20" | undefined for (Paginated & AddressDetailsEthereumTxids).tokens.type, but got: undefined

      127 |       return value as T
      128 |     }
    > 129 |     return assertType(codec, value, ...rest)
          |            ^
      130 |   }
      131 |
      132 |
```

Reply:
@lexy212 I looked at the raw response we're receiving and couldn't find any token where `type` is not `'ERC20'` using (`tokens.find((t) => t.type !== 'ERC20')`).

Are you able to identify that the data is in fact bad or is there some other sneaky bug causing this?